src/main/scala/play-json.scala

```diff
@@ -47,16 +47,15 @@ package object internals {
   }
 
   import scala.collection._
-  import scala.collection.generic.CanBuildFrom
-  private[json] implicit class TraversableLikeExtensions[A, Repr]( val coll: TraversableLike[A, Repr] ) extends AnyVal {
+  private[json] implicit class IterableExtensions[A, Repr]( val coll: Iterable[A] ) extends AnyVal {
     /** Eliminates duplicates based on the given equivalence function.
      *  There is no guarantee which elements stay in case element two elements are considered equivalent.
      *  this has runtime O(n^2)
      *  @param symmetric comparison function which tests whether the two arguments are considered equivalent.
      */
-    def distinctWith[That]( equivalent: ( A, A ) => Boolean )( implicit bf: CanBuildFrom[Repr, A, That] ): That = {
```

Comment:
Instead of creating a different version of this method for scala 2.13 I removed the `CanBuildFrom` and return an Iterable...